acm/odh-edge/base/namespace.yaml

```diff
@@ -3,3 +3,5 @@ kind: Namespace
 metadata:
   # edit this to the namespace where the model is deployed
   name: model-to-edge
+  labels:
+    argocd.argoproj.io/managed-by: openshift-gitops
```

Comment:
@LaVLaS this is the label that I was saying would need to be updated to have a different value if trying to use it with upstream ArgoCD. I'd have thought it would be the ArgoCD instance name that would be the value, but this page seems to indicate that it's the _namespace_ that ArgoCD is in (I'm not sure if that's correct): https://docs.openshift.com/gitops/1.11/argocd_instance/setting-up-argocd-instance.html

In any case, I think probably in the upstream case, the ArgoCD instance and namespace are probably both called `argocd`, and with OpenShift GitOps, they're both called `openshift-gitops`.

Reply:
Ok. So for right now, we just need to be aware that this needs to change if someone is using upstream ArgoCD?

If so, can we add comment just as a reminder?  Otherwise it's LGTM

Reply:
> Ok. So for right now, we just need to be aware that this needs to change if someone is using upstream ArgoCD?

Yes :+1: 

> If so, can we add comment just as a reminder? Otherwise it's LGTM

Updated, thanks!